frontend/src/components/TriggeredPrompt.tsx

```diff
@@ -93,26 +99,28 @@ const TriggeredPrompt: FunctionComponent<TriggeredPromptProps> = ({
     }
 
     // Intervention activities
-    getDuration(prompt2, 0, 50, updatePrompt2);
-    if (canPlayPrompt1 && canPlayPrompt2) {
-      // Activities 1 and 4 do not set triggerSecondPrompt, but do set promptDelay
-      if (promptDelay !== undefined) {
-        playPrompt1();
-        setTimeout(() => {
-          playPrompt2();
-          triggerSecondPrompt = true;
-        }, prompt1Duration * 1000 + promptDelay);
-        return () => {
+    if (prompt2Url !== undefined) {
+      getDuration(prompt2, 0, 50, updatePrompt2);
+      if (canPlayPrompt1 && canPlayPrompt2) {
+        // Activities 1 and 4 do not set triggerSecondPrompt, but do set promptDelay
+        if (promptDelay !== undefined) {
+          playPrompt1();
+          setTimeout(() => {
+            playPrompt2();
+            triggerSecondPrompt = true;
```

Comment:
What does this do? I think on rerender this will be ignored, so will this work without changing this value?

Reply:
This is set so that the replay button replays the audio from prompt2 instead of prompt1, after prompt2 has actually been played.

Reply:
Jkjk I'm bugging, I think you're right. lemme do some testing